package.json

```diff
@@ -30,6 +30,7 @@
     "prepare": "husky install",
     "pre-commit": "lint-staged",
     "version": "yarn install --mode=skip-build && git add yarn.lock yarn-project.nix",
+    "patch-sibling-deps-versions": "node scripts/patch-versions/index.js",
```

Comment:
This one has left from the original commit and should be removed since the script is not required

Reply:
Nice catch, squashed to e83ac7f5bcf90aa3707076a87fc3516d295f6edd